wesnoth-ei.po

```diff
@@ -2907,11 +2921,12 @@ msgid ""
 "privately. None know what was said, but when he came out he announced that "
 "the seer was dead, and he had... chosen... me as his new advisor."
 msgstr ""
-"V celé krajině byli jen dva mágové světla, kteří převyšovali všechny "
-"ostatní. Mág z východu jménem Ravan a já. Král si chtěl vybrat toho "
-"nejlepšího rádce, proto nás oba poslal za Geldrem. Pak si s vidoucím "
-"promluvil o samotě. O čem si povídali, to nikdo neví, ale když se vrátil, "
-"ohlásil, že vidoucí je mrtev a vybral si mne jako svého nového rádce."
+"Král se proto rozhodl vyhledat takového vynikajícího učence. V celé zemi "
+"byli dva mágové, kteří jasně vyčnívali nad ostatními. Já a můj dobrý přítel "
+"Ravan, mág z východu. Král si přál vybrat nejlepšího rádce, a tak nás oba "
+"poslal před Galdrena. Pak si s věštcem promluvil v soukromí. Nikdo neví, o "
+"čem si povídali, ale když král vyšel ven, oznámil, že věštec je mrtvý a že "
```

Comment:
Prosím, překládej seer jako vidoucí. Už když jsem to v roce 2019 překládal, věděl jsem, proč zrovna takhle. A nedávno, když jsem pracoval na nových ještěráckých jednotkách, jsem si dělal novou rešerši a potvrdil si to. Seer stojí nad jasnovidci nebo věštci, převyšuje je, vidí dál, vidoucích není tolik, jsou výjimečnější. To vše, mám za to, má to slovo znamenat.
```suggestion
"poslal před Galdrena. Pak si s vidoucím promluvil v soukromí. Nikdo neví, o "
"čem si povídali, ale když král vyšel ven, oznámil, že vidoucí je mrtvý a že "
```

Reply:
OK. Piš to prosím někam do konvencí nebo pravidel.

Reply:
Doplněno do [Pravidel](https://wiki.wesnoth.org/CS:Pravidla#Osoby).